vpr/src/route/parallel_connection_router.cpp

```diff
@@ -750,13 +833,13 @@ void ParallelConnectionRouter::empty_heap_annotating_node_route_inf() {
     //the cost of all nodes considered by the router (e.g. nodes never
     //expanded, such as parts of the initial route tree far from the
     //target).
-    pq_node_t tmp;
-    while (heap_.try_pop(tmp)) {
-        rr_node_route_inf_[tmp.index].path_cost = tmp.cost;
-        rr_node_route_inf_[tmp.index].backward_path_cost = tmp.backward_path_cost;
-        // Push back serially at this point
-        modified_rr_node_inf_[modified_rr_node_inf_.size()-1].push_back(tmp.index);
-    }
+    // pq_node_t tmp;
```

Comment:
Its a very good idea to remove this function. Can you add an error here so it would crash if this function is ever called?